hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsHttpOperation.java

```diff
@@ -110,6 +115,10 @@ protected  HttpURLConnection getConnection() {
     return connection;
   }
 
+  void setConnection(final HttpURLConnection connection) {
+    this.connection = connection;
+  }
+
```

Comment:
As suggested in TestAbfsClient, lets remove it.

Reply:
Kindly refer to https://github.com/pranavsaxena-microsoft/hadoop/commit/ff0a272c525437b8d7ae6dcaf74419139cf2c1fe and https://github.com/pranavsaxena-microsoft/hadoop/commit/443c295e2e4d27155054b4823dd379a0c2c5517e on the approach to remove this setter.